test/models/actions-inputs-model.test.js

```diff
@@ -1,61 +1,79 @@
   test('if error provided, should populate errorMessage', () => {
     const error = 'sample error message'
-    const { model } = actionsInputsModel('', {}, error)
+    const { model } = actionsInputsModel('SD74445738', actions.fg6, error)
     expect(model.errorMessage).toEqual(
       expect.objectContaining({ text: error })
     )
   })
 
   test('if error isn\'t provided, errorMessage should be omitted', () => {
-    const { model } = actionsInputsModel('', {})
+    const { model } = actionsInputsModel('SD74445738', actions.fg6)
     expect(model.errorMessage).toBeUndefined()
   })
 
   test('label class set to govuk-label--xl', () => {
-    const { model } = actionsInputsModel('', {})
+    const { model } = actionsInputsModel('SD74445738', actions.fg6)
     expect(model.label.classes).toBe('govuk-label--xl')
   })
 
   test('label isPageHeading is true', () => {
-    const { model } = actionsInputsModel('', {})
+    const { model } = actionsInputsModel('SD74445738', actions.fg6)
     expect(model.label.isPageHeading).toBeTruthy()
   })
 
   test('id is actionInput', () => {
-    const { model } = actionsInputsModel('', {})
+    const { model } = actionsInputsModel('SD74445738', actions.fg6)
     expect(model.id).toBe('actionInput')
   })
 
   test('name is actionInput', () => {
-    const { model } = actionsInputsModel('', {})
+    const { model } = actionsInputsModel('SD74445738', actions.fg6)
     expect(model.label.classes).toBe('govuk-label--xl')
   })
 
   test('title and hint refers to fencing for Action ID FG1', () => {
     const parcelRef = 'AB1234'
-    const { model } = actionsInputsModel(parcelRef, { id: 'FG1' })
-    expect(model.label.text).toBe('Enter a fence length in metres')
-    expect(model.hint.text.trim()).toBe(`How long is the fence you want to claim on in metres for ${parcelRef}?`)
+    const { model } = actionsInputsModel(parcelRef, actions.fg6)
+    expect(model.label.text).toBe('Enter the fence length in metres')
+    expect(model.hint.text.trim()).toBe(`What is the fence length you wish to claim for in metres for ${parcelRef}?`)
   })
 
   test('title and hint refers to acres for Action ID SW6', () => {
     const parcelRef = 'AB1234'
-    const { model } = actionsInputsModel(parcelRef, { id: 'SW6' })
-    expect(model.label.text).toBe('Enter the area in hectares')
-    expect(model.hint.text.trim()).toBe(`What is the area you wish to claim on in hectares for ${parcelRef}?`)
+    const { model } = actionsInputsModel(parcelRef, actions.sw6)
+    expect(model.label.text).toBe('Enter the winter crops area in hectares')
+    expect(model.hint.text.trim()).toBe(`What is the winter crops area you wish to claim for in hectares for ${parcelRef}?`)
   })
 
   test('Parcel data is included in the model when a parcel is included', () => {
-    const { parcelModel } = actionsInputsModel('AAA', { id: 'FG1' }, '', allParcels)
+    const parcelModel = actionsInputsModel('SD74445738', actions.fg6, '', allParcels).parcelModel
```

Comment:
Following the convention above where we're using object destructuring, you can do:
```
const { parcelModel } = actionsInputsModel('SD74445738', actions.fg6, '', allParcels)
```

Reply:
Good spot.